Islandora/islandora_internet_archive_bookreader
# ISLANDORA-2023: modified javascript so that it will update the page s…

**JIRA Ticket**: (link)
https://jira.duraspace.org/browse/ISLANDORA-2023 

* Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)

# What does this Pull Request do?
This will modify the javascript in the module so that the share link to "Link to this page view:" will be updated when the user navigates to a different page.

A brief description of what the intended result of the PR will be and/or what problem it solves.
The end result should be such that the "Link to this page view:" that is created will reflect the current page -- regardless of how the user navigates to a different page.

# What's new?
A in-depth description of the changes made by this PR. Technical details and possible side effects.
It takes the current location and modifies it using the existing javascript functions in the same unit to update the #pageview element (that is in the hidden "share dialog").

# How should this be tested?
Navigate to a book, click the share button to see what the value for "Link to this page view:" is.  Navigate to that URL to see if it is the correct page.  Navigate to a different page with the pointer or the slide control -- and repeat the steps to confirm that the new "Link to this page view:" points to the new page.

A description of what steps someone could take to:
* Reproduce the problem you are fixing (if applicable)
* Test that the Pull Request does what is intended.
* Please be as detailed as possible.
* Good testing instructions help get your PR completed faster.


# Additional Notes:
It may be helpful to update documentation so that the feature to share a page is mentioned.

# Interested parties
none that I can think of currently, but I am sure that anybody who uses this module would expect that the share link would give you a link to the currently viewed page.
